src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -591,7 +591,7 @@ private OverallStatus getOverallStatusBasedOnServiceState(OverallStatus overallS
     private DeploymentInformation getDeploymentInformation(Map<String, Object> deploymentDetails) {
         DeploymentInformation deploymentInformation = DeploymentInformation.builder()
                 .status((String) deploymentDetails.get(DEPLOYMENT_STATUS_KEY_NAME))
-                .deploymentId((String) deploymentDetails.get(DEPLOYMENT_ID_KEY_NAME))
+                .deploymentId((String) deploymentDetails.get(GG_DEPLOYMENT_ID_KEY_NAME))
```

Comment:
Add a comment here why your need to set the gg deployment id and not the old deployment id

Reply:
will add

Reply:
let's remove the part about previous deployment id - once you check this in you won't see the the diff and won't know what the 'previous' deployment id is

Reply:
sure